#
# old_revision [010297e01bc32090e5a7de8f5bd1c4c69d72f532]
#
# patch "foo"
#  from [83f5a5c359f3dc8317519240e32f1f51f68bc051]
#    to [da39a3ee5e6b4b0d3255bfef95601890afd80709]
#
============================================================
--- foo	83f5a5c359f3dc8317519240e32f1f51f68bc051
+++ foo	da39a3ee5e6b4b0d3255bfef95601890afd80709
@@ -1,2 +0,0 @@
-1
-2
